### ISSUE TITLE
tsc cannot import type declaration files.

### ISSUE THREAD

**Issue by gkswjdzz** [vvo/next-iron-session#346]
When I compile lib folder with tsc, error is occurred like below.

```
node_modules/next-iron-session/lib/index.d.ts(1,45): 
error TS6137: Cannot import type declaration files. Consider importing 'cookie' instead of '@types/cookie'.
```

This is my tsconfig.json.
```
{
  "compilerOptions": {
    "module": "commonjs",
    "moduleResolution": "node",
    "target": "ES2019",
    "lib": ["dom", "es2019"],
    "outDir": "dist",
    "rootDir": "src",
    "declaration": true,
    "sourceMap": true,
    "experimentalDecorators": true,
    "emitDecoratorMetadata": true,
    "strict": true,
    "noImplicitAny": false,
    "noImplicitThis": true,
    "noImplicitReturns": true,
    "strictNullChecks": true,
    "esModuleInterop": true,
    "strictPropertyInitialization": false,
    "resolveJsonModule": true
  },
  "include": ["src/**/*.ts"],
  "exclude": [
    "node_modules",
    "dist",
  ]
}
```

I can add 'skipLibCheck' option to `compilerOptions`, I hesitate to use this method.

Is there any way that can pass compile code without `skipLibCheck`, OR anything I am missing?

**Comment by vvo** [comment]
Thanks! Just saw that @types is automatically added https://www.typescriptlang.org/tsconfig#typeRoots

**Comment by vvo** [comment]
fixed in 4.2.0, you can upgrade now, thanks for reporting!
